docs/source/joblib.rst

```diff
@@ -52,6 +52,15 @@ validated parameter search as follows.
    with parallel_backend('dask.distributed', scheduler_host='localhost:8786'):
        search.fit(digits.data, digits.target)
 
+## Nested Parallelism
+
+Note that this code will not work if you distribute it via dask in order to implement nested parallelism. The reason is that each worker will have a fresh copy of sklearn and `from sklearn.externals.joblib import parallel_backend` will not apply to them, resulting in a "KeyError" when trying to distribute, but no problem when running the inner loop locally. The solution is 
+
+.. code-block:: python
+
+    from distributed.joblib import DistributedBackend 
+    register_parallel_backend('distributed', DistributedBackend, make_default=True)
```

Comment:
Yes, that does make sense.  It wasn't clear to me from the documentation that is written here though.  Also, in that case I would recommend just importing `distributed.joblib` which does the operation you suggest, but will track the version of dask a bit better.

As far as I understand though this isn't common case behavior, so we would probably want to provide some context (and maybe give general documentation on that context) before diving into this point.

Maybe a general warning would be useful about how you have to `import distributed.joblib` regardless?  And that this is even true if you choose to call Joblib from inside a remote task, in which case you should probably `import distributed.joblib` within your function.